userspace/libsinsp/filterchecks.cpp

```diff
@@ -2798,20 +2798,20 @@ bool sinsp_filter_check_thread::compare(sinsp_evt *evt)
 ///////////////////////////////////////////////////////////////////////////////
 const filtercheck_field_info sinsp_filter_check_gen_event_fields[] =
 {
-	{PT_UINT64, EPF_NONE, PF_ID, "evt.num", "event number."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.time", "event timestamp as a time string that includes the nanosecond part."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.time.s", "event timestamp as a time string with no nanoseconds."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.time.iso8601", "event timestamp in ISO 8601 format, including nanoseconds and time zone offset (in UTC)."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.datetime", "event timestamp as a time string that includes the date."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.datetime.s", "event timestamp as a datetime string with no nanoseconds."},
-	{PT_ABSTIME, EPF_NONE, PF_DEC, "evt.rawtime", "absolute event timestamp, i.e. nanoseconds from epoch."},
-	{PT_ABSTIME, EPF_NONE, PF_DEC, "evt.rawtime.s", "integer part of the event timestamp (e.g. seconds since epoch)."},
-	{PT_ABSTIME, EPF_NONE, PF_10_PADDED_DEC, "evt.rawtime.ns", "fractional part of the absolute event timestamp."},
-	{PT_RELTIME, EPF_NONE, PF_10_PADDED_DEC, "evt.reltime", "number of nanoseconds from the beginning of the capture."},
-	{PT_RELTIME, EPF_NONE, PF_DEC, "evt.reltime.s", "number of seconds from the beginning of the capture."},
-	{PT_RELTIME, EPF_NONE, PF_10_PADDED_DEC, "evt.reltime.ns", "fractional part (in ns) of the time from the beginning of the capture."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.pluginname", "if the event comes from a plugin, the name of the plugin that generated it. The plugin must be currently loaded."},
-	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.plugininfo", "if the event comes from a plugin, a summary of the event as formatted by the plugin. The plugin must be currently loaded."},
+	{PT_UINT64, EPF_NONE, PF_ID, "evt.num", "Event Number", "event number."},
+	{PT_CHARBUF, EPF_NONE, PF_NA, "evt.time", "Time", "event timestamp as a time string that includes the nanosecond part."},
```

Comment:
What about prepending `Event` to all fields here?

Reply:
I mean, i'd always add the filter category as a prefix to display name, for below fields too; wdyt?

Reply:
I'm not sure; this is kind of a fuzzy UI area. If the fields will be shown without any other context, then prefixing the category makes sense. If fields are grouped together the category can become repetitive.

We typically don't do this in Wireshark because the primary widget for displaying fields (the packet detail) groups all of the items for a given protocol together. For example, in the output below it's apparent that all of the items are IPv4 fields, so we don't prepend them with "IP" or "IPv4".

```
Internet Protocol Version 4, Src: 207.183.142.87 (207.183.142.87), Dst: 206.65.9
8.18 (206.65.98.18)
    0100 .... = Version: 4
    .... 0101 = Header Length: 20 bytes (5)
    Differentiated Services Field: 0x10 (DSCP: Unknown, ECN: Not-ECT)
        0001 00.. = Differentiated Services Codepoint: Unknown (4)
        .... ..00 = Explicit Congestion Notification: Not ECN-Capable Transport 
(0)
    Total Length: 60
    Identification: 0x0ae1 (2785)
    Flags: 0x40, Don't fragment
        0... .... = Reserved bit: Not set
        .1.. .... = Don't fragment: Set
        ..0. .... = More fragments: Not set
    ...0 0000 0000 0000 = Fragment Offset: 0
    Time to Live: 64
    Protocol: TCP (6)
    Header Checksum: 0xa168 [validation disabled]
    [Header checksum status: Unverified]
    Source Address: 207.183.142.87 (207.183.142.87)
    Destination Address: 206.65.98.18 (206.65.98.18)
```

I'm guessing that the fields will most often be grouped together. If it's OK with you I'd like to leave the prefixes out for now.